Fixing inconsistencies and updating instructions for the Installations.md file.

JaytidaA opened the pull request:
Resolves issue #148 
* Updated the pics for steps 7, 8, 9 and esp_idf.jpeg
* Changes to the git clone instructions for MacOS
* Changed the initial Windows Project to be built and compiled to 1_led_blink instead of hello_world

Shankari02 commented:
@JaytidaA please make the required changes.

JaytidaA commented on Installations.md:
```diff
@@ -105,19 +105,17 @@ sudo chmod +x wall_e_install.sh
-### STEP 3 : Installing the Pre-Requisites 
-Copy this command in the command in the terminal , then press return 
+### STEP 2 : Installing the necessary file
+- Download the driver by executing the following command in the terminal.
```
That is not the installation for the driver but rather the curling of the raw installation script from the Github repository. I have made changes to better reflect this and prevent confusion in the latest commit.

Shankari02 replied:
Cool, its looks fine now

JaytidaA commented on Installations.md:
```diff
@@ -105,19 +105,17 @@ sudo chmod +x wall_e_install.sh
-- Download the driver by clicking on this [link](https://www.silabs.com/documents/public/software/Mac_OSX_VCP_Driver.zip) , unzip it, then install it on the system
-
-### STEP 2 : Opening The Terminal
+### STEP 1 : Opening The Terminal
```
I have used curl instead to maintain consistency.

Shankari02 replied:
Fine!

Shankari02 commented:
@JaytidaA 
![image](https://github.com/user-attachments/assets/c6675554-b1ce-48cf-85d3-f14dc2b0c860)

![image](https://github.com/user-attachments/assets/a00f6fb9-c5d4-4de7-8e53-47000282e12e)

This is what I had told to remove, because we are currently using CH340 usb-uart which doesn't require explicit installation

JaytidaA commented:
I have made the necessary changes:
+ Removed the driver installation instructions
+ Removed the related pictures as they are no longer needed
+ Renamed 18.jpg to 17.jpg to maintain sequencing.

Shankari02 commented on Installations.md:
```diff
@@ -182,26 +182,14 @@ Connect your ESP32 board to the computer and check under what serial port the bo
-<p align="center">
-  <img src="./documentation/Assets/Installation/19.jpeg">
-</p> 
+The USB to UART bridge will be automatically detected after a few seconds of plugging it to your machine.
```
Looks good!

Shankari02 reviewed (approved):
LGTM!
